apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -0,0 +1,59 @@
+/* global $ */
+var React = require('react');
+
+var colorPicker = require('../colpick');
+
+var PropertyRow = React.createClass({
+  propTypes: {
+    initialValue: React.PropTypes.string.isRequired,
+    handleChange: React.PropTypes.func
+  },
+
+  getInitialState: function () {
+    return {
+      value: this.props.initialValue
+    };
+  },
+
+  componentDidMount: function () {
+    var element = this.getDOMNode().querySelector('.colorPicker');
+    $(element).colpick({
```

Comment:
I think we might want to do this on componentDidUpdate as well. otherwise the color picker might get clobbered if the component ever updates.